### PR TITLE
OCPBUGS-37107: Power VS: Disable SNAT when specifying imageContentSources

### PR DESCRIPTION
**Pull request by mjturek**
When doing a disconnected deploy we'll need to disable SNAT

### CODE REVIEW

File: pkg/asset/manifests/powervs/cluster.go [hunk 1]
```diff
@@ -175,7 +175,12 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 			return nil, fmt.Errorf("unable to find a DNS server for specified VPC: %s %w", installConfig.Config.PowerVS.VPCName, err)
 		}
 
-		powerVSCluster.Spec.DHCPServer.DNSServer = &dnsServerIP
+		powerVSCluster.Spec.DHCPServer = &capibm.DHCPServer{
+			Cidr: &dhcpSubnet,
```
Comment (barbacbd) [inline pkg/asset/manifests/powervs/cluster.go:179]: Nit: Can we add a comment about adding the subnet information here too for commit purposes.

Comment (mjturek) [reply pkg/asset/manifests/powervs/cluster.go:179]: @barbacbd this is actually so I use the same subnet specified above but this makes me realize that I went about this wrong. Reinstantiating the struct is not the right move. I should update it if possible.

Will recommit in a moment.

**Comment by AshwinHIBM** [comment]
`golint` says `pkg/asset/manifests/powervs/cluster.go:182:2: unnecessary trailing newline (whitespace)`
Other than that /lgtm

**Review by r4f4 — COMMENTED** [review]
/approve

**Pull request merged**